### ISSUE TITLE
Load Content and Choose File buttons don't go away in fullscreen

### ISSUE THREAD

**Issue by ooiikkjj** [BenTearzz/RemarkPortable#4]
Load Content and Choose File buttons don't go away in fullscreen

**Comment by benstigsen** [comment]
What browser are you using?
I cannot reproduce this with Firefox 84.0.2

**Comment by ooiikkjj** [comment]
Edge...

BTW: so glad to see Display will be light and nimble...

Some wishlist items there likely to be popular: markdown, MathJax or liter, display an xxx.md file like RemarkPortable does instead of tinkering with html, ...

**Comment by benstigsen** [comment]
Yep, I've thought all of those things @ooiikkjj, but keeping it lightweight is the challenge. MathJax (2.7) adds ~62 KB. Although, 64 KB is still ~11x smaller than RemarkPortable.

I'm thinking about keeping modules as comments that can be commented out in the HTML document.
So that the comments can be removed to activate it:
```html
<!-- MathJax support -->
<!--<script src="./assets/js/mathjax.min.js?config=TeX-MML-AM_CHTML"></script>-->
```

But I'm not sure what people think when I say lightweight. Is this based on what the HTML document **_requires_** to work, or is this based on how much is downloaded?

**Comment by ooiikkjj** [comment]
ok...thanks...

(minislides by ThomasR also takes a code-golfing philosophy and has a very tiny footprint)

**Comment by benstigsen** [comment]
I did not know about that, but it looks great! It sure is tiny, and it even supports showing things incrementally, although there are several things I would change about it.

- I would predefine some slide transitions (this would be more CSS but easier to make each slide behave a certain way)
- Markdown support
- MathJax support
- Easier styling (instead of having to use raw CSS right in the HTML document or a new CSS document)

**Comment by ooiikkjj** [comment]
some wishlist items, taken from RemarkPortable...

1. an ability to 'load' an .md file...
2. simple CSS choices, perhaps from a list: e.g., font:color, slidebackground:color, etc...
   here's a thought: the --- separator for slides in .md could be embellished to this ---ab--- where a represents one of, say, 10 font choices, and b represents, say, one of 12 background choices (like color or image), all in the CSS file so there's no cruft in the actual file.
3. offer not just 4:3, but also 16:9 aspects
4. fonts/content to autoscale for fullscreen
5. the ---xy--- format can be infinitely adaptable but won't make the .md file look terrible while allowing a lot of choice through the CSS file...it could be even done on a per-slide basis

i don't know any programming (or html, or CSS, or such) so these may be not simple wishlists...

thanks for reading

**Comment by benstigsen** [comment]
They are good suggestions, with the most important ones being number 3 and 4.
One I'm not sure about though, is the `---ab---` system. If it's going to follow regular markdown style is should follow the CommonMark style. But to make it even more lightweight, I might just make my own tiny markdown flavor instead, which would also allow me to implement what you suggested.

**Comment by ooiikkjj** [comment]
a suggestion for a new, tiny markdown would be that column 1 is reserved for 'instructions'...so the following conventions could work:

# new slide
! big heading (e.g., h1)
@ smaller heading (like h2)
* list item
> incremental list
$ url (like in an iframe...could be image, pdf, youtube, website, etc.)
 (i.e., blank) would continue previous format

some 'meta-commands' could be

f fullscreen
n notes screen
a annotation
b black
w white screen

etc...Of course, *you* would be making the choices and these are just illustrations of what a markdown 'first-column for instructions' would look like.

am able to do most of what i want by just using <iframe> in RemarkPortable, or Display, or Presenta, or minislides...or even, in markdeep...etc.

thanks for reading.

**Comment by ooiikkjj** [comment]
sorry...my suggestions got taken as markup, so it doens't read right...
meant:

`# new slide
`! h1
`@ h2
`* list item
`> incremental list
`# url (for image, site, youtube, pdf, etc, including local files)
`_ (blank) would continue previous instruction

**Comment by ooiikkjj** [comment]
still not right, but you get the idea

**Comment by benstigsen** [comment]
I think I'll keep it closer to regular markdown, but allow the following:
```markdown
# h1
## h2

*italic*
**bold**
***bold and italic***

~~strikethrough~~

- list
- elements

[link here](www.stigsen.xyz)
```
So only not all title sizes are allowed, and underscore is not used used for making things bold or italic.

**Comment by ooiikkjj** [comment]
please keep the load xyz.md file option as that cleanly separates the content from the presenting engine from the styling...
(should this conversation move to Display issues, as i think it's getting outside of RemarkPortable now...maybe doesn't matter)

thanks for listening/responding...

**Comment by benstigsen** [comment]
> should this conversation move to Display issues, as i think it's getting outside of RemarkPortable now...maybe doesn't matter

Yeah probably.

**Comment by ooiikkjj** [comment]
oh no...is Display dead? hibernating?
hope it comes back!

**Comment by benstigsen** [comment]
I've made Display private for now since I have a lot of unfinished projects that have no reason to be public. 
I want to continue working on Display but I'm currently very busy with assignments and I'm working on another project at the moment.
